packages/shared/src/types/waitlist.ts

```diff
@@ -1,7 +1,38 @@
+import type { InvitationResource } from './invitation';
 import type { ClerkResource } from './resource';
 
 export interface WaitlistResource extends ClerkResource {
+  /**
+   * The unique identifier for the `WaitlistResource`.
+   */
   id: string;
-  createdAt: Date | null;
-  updatedAt: Date | null;
+  /**
+   * The email address associated with the `WaitlistResource`.
+   */
+  emailAddress: string;
```

Comment:
This file is the FAPI representation of a WaitlistEntry, which is just

```
{
    "object": "waitlist_entry",
    "id": "wle_35nCTK0zjvUt181oXWwlVvd3aTO"
}
```

I think the fields you've added here are from the BAPI representation? `backend` uses https://github.com/clerk/javascript/blob/69833fefe7a2f2eb0eb72d94617fe39a208ed310/packages/backend/src/api/resources/WaitlistEntry.ts which has these fields (albeit without the comments)